core/src/main/scala/org/apache/spark/h2o/utils/H2OContextRestAPIUtils.scala

```diff
@@ -52,6 +52,22 @@ trait H2OContextRestAPIUtils extends H2OContextUtils {
     getNodes(cloudV3)
   }
 
+  def verifyWebOpen(nodes: Array[NodeDesc], conf: H2OConf): Unit = {
+    val nodesWithoutWeb = nodes.flatMap { node =>
+      try {
+        getCloudInfoFromNode(node, conf)
+        None
+      } catch {
+        case _: H2OClusterNodeNotReachableException => Some(node)
+      }
+    }
+    if (nodesWithoutWeb.nonEmpty) {
+      throw new H2OClusterNodeNotReachableException(
+        s"The following worker nodes are not reachable, but belong to the cluster identified by " +
+          s"${conf.h2oCluster.get}:${nodesWithoutWeb.mkString("\n")}", null)
```

Comment:
Could it happen that `conf.h2oCluster` is `None`?

Reply:
In this case no, in manual cluster mode the h2oCluster needs to be always defined